core/src/main/java/ai/timefold/solver/core/impl/phase/AbstractPhase.java

```diff
@@ -77,6 +79,11 @@ public boolean isAssertShadowVariablesAreNotStaleAfterStep() {
 
     public abstract String getPhaseTypeString();
 
+    @Override
+    public boolean isInitializationPhase() {
```

Comment:
Arguably this needs a better name. What does it mean to be an "initialization phase"?

I'd be more literal here. I'd call this "triggersFirstInitializedSolutionEvent()". Javadoc can then reference the definition of what that event is.